moto/transcribe/__init__.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
+
+from .models import transcribe_backends
+
+transcribe_backend = transcribe_backends["us-east-1"]
+mock_transcribe = transcribe_backend.decorator
+mock_transcribe_deprecated = transcribe_backend.deprecated_decorator
```

Comment:
We're slowly trying to move away from the deprecated decorators - let's just stick to the normal one

Reply:
Done.